packages/react-cards/src/components/Card/Card.styles.ts

```diff
@@ -1,4 +1,4 @@
-import { getGlobalClassNames, HighContrastSelector } from '@uifabric/styling';
+import { getGlobalClassNames, HighContrastSelector } from 'office-ui-fabric-react';
```

Comment:
You might want to import these from `oufr/lib/Styling` to reduce bundle size impact for consumers. (In theory there should be no impact, but with some bundling setups it can make a difference.)

Same in other places.

Reply:
Actually never mind, since you're already importing from the root elsewhere it's probably fine to just leave this as-is.